README.md

```diff
@@ -2,6 +2,89 @@
 Docker image for universal postgres backups
 
 # Roadmap
-- [ ] Add support for S3
+- [X] Add support for S3
+- [X] Add CI/CD to publish image to DockerHub
 - [ ] Add retention policy settings by env vars
-- [ ] Notify about backup status by HTTP-request
\ No newline at end of file
+- [ ] Notify about backup status by HTTP-request
+- [ ] Add docker-compose example
+
+## Docker build
+```shell
+docker build . -t numdes/nd_postgres_backup:v*.*.*
+```
+
+# Usage
+## Backup manually:
+```shell
+docker run --rm -it \
+    -e POSTGRES_HOST="FQDN-OR-IP" \
+    -e POSTGRES_DB="DB-NAME" \
+    -e POSTGRES_USER="DB-USER" \
+    -e POSTGRES_PASSWORD="PASS" \
+    -e S3_ENDPOINT=http://YOUR-S3 \
+    -e S3_ACCESS_KEY_ID="KEY-ID" \
+    -e S3_SECRET_ACCESS_KEY="KEY-SECRET" \
+    -e S3_BUCKET="BUCKET-NAME" \
+    -e NOTIFICATION_URL=http://webhook \
+    -e TELEGRAM_CHAT_ID=point_to_notify_group \
+    -e POSTGRES_PORT=if_not_5432 \
+    --entrypoint /bin/bash \
```

Comment:
Why do we need to provide entrypoint here?

Reply:
As it is an example of manual backup creation (like for testing purposes) we need to substitute image's `ENTRYPOINT` to one which will open console for us.